src/resources/MachineLearning/ModelListing/ModelListingConfiguration.ts

```diff
@@ -0,0 +1,24 @@
+export interface MLListingModel {
+    modelDisplayName: string;
+    modelId: string;
+    engineId: string;
+    estimatedPreviousModelUpdateTime: number;
+    nextModelUpdateTime: number;
+    platformVersion: 1 | 2;
+    modelSizeStatistic: number;
+    readyForAssociation: boolean;
+    modelStatusInfo: MLModelStatusInfo;
+    modelAssociations: MLModelAssociation[];
+}
+
+export interface MLModelStatusInfo {
+    modelStatus: string;
```

Comment:
We can be more precise here

```suggestion
    modelStatus: 'ARCHIVED' | 'SOON_TO_BE_ARCHIVED' | 'BUILD_IN_PROGRESS' | 'ERROR' | 'ERROR_INTERNAL' | 'LIMITED' | 'NOT_ASSOCIATED' | 'ACTIVE' | 'INACTIVE';
```

Reply:
Yes! I put this as string instead of enum just in case a status is added in future. But i can revisit this part